src/platform/user/profile/vet360/components/PhoneField/PhoneEditModal.jsx

```diff
@@ -94,23 +94,28 @@ class PhoneEditModal extends React.Component {
           international number, please check back later.
         </p>
       </AlertBox>
-
-      <PhoneTextInput
-        additionalClass="usa-only-phone"
-        label="Number"
-        charMax={14}
-        required
-        field={{ value: this.props.field.value.inputPhoneNumber, dirty: false }}
-        onValueChange={this.onChange('inputPhoneNumber')}
-        errorMessage={this.props.field.validations.inputPhoneNumber}
-      />
-
-      <ErrorableTextInput
-        label="Extension"
-        charMax={10}
-        field={{ value: this.props.field.value.extension, dirty: false }}
-        onValueChange={this.onChange('extension')}
-      />
+      <div className="vads-u-padding-y--1p5">
```

Comment:
Added wrapping div as the formation-react component does not allow for passing a className that would end up on the wrapping component within ErrorableTextInput.  This will work for now.